### PR TITLE
Right-align maximum entitlement weights and dollar amounts on SSW

### PR DESCRIPTION
**Pull request by mkrump**
## Description

Update pdf generation code to allow for text alignment. Then right align various numeric fields on the Shipment Summary Worksheet.

## Setup

`make server_test`

### Test in the office client
`make server_run`
`make office_client_run`
1) Log into the office app and select a move with a PPM
2) Click Create Paperwork -> Click Download worksheet

### Test with cmd line tool
` go run cmd/generate_shipment_summary/main.go -move 0db80bd6-de75-439e-bf89-deaafa1d0dc8 -debug`

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2181745/stories/163805748) 

## Screenshots

screen shot 2019-02-22 at 12 44 40 pm

### CODE REVIEW

File: pkg/paperwork/forms.go [hunk 1]
```diff
@@ -246,7 +256,12 @@ func (f *FormFiller) drawData(fields map[string]FieldPos, data interface{}) erro
 			tempLineHeight = *formField.lineHeight
 		}
 
-		f.pdf.MultiCell(formField.width, tempLineHeight, displayValue, "", "", false)
+		fieldAlignment := alignment
+		if formField.alignStr != nil {
+			fieldAlignment = *formField.alignStr
+		}
+
+		f.pdf.MultiCell(formField.width, tempLineHeight, displayValue, "", fieldAlignment, false)
```
Comment (Ronolibert) [inline pkg/paperwork/forms.go]: Nice! So the library does have it built in

File: pkg/paperwork/shipment_summary_worksheet.go [hunk 2]
```diff
@@ -7,29 +7,29 @@ var ShipmentSummaryPage1Layout = FormLayout{
...
-		"ShipmentPickUpDates":             FormField(54, 122.5, 46, floatPtr(10), nil),
-		"ShipmentWeights":                 FormField(103, 122.5, 41, floatPtr(10), nil),
-		"ShipmentCurrentShipmentStatuses": FormField(153.5, 122.5, 41, floatPtr(10), nil),
-		"GCC100":                          FormField(40, 182.5, 22, floatPtr(10), nil),
-		"TotalWeightAllotmentRepeat":      FormField(74, 182.5, 16, floatPtr(10), nil),
-		"GCC95":                           FormField(40, 188.5, 22, floatPtr(10), nil),
-		"GCCMaxAdvance":                   FormField(40, 201, 22, floatPtr(10), nil),
+		"PreparationDate":                 FormField(155.5, 22, 46, floatPtr(10), nil, nil),
+		"ServiceMemberName":               FormField(10, 42, 105, floatPtr(10), nil, nil),
+		"WeightAllotment":                 FormField(74, 92, 16, floatPtr(10), nil, stringPtr("R")),
```
Comment (kahlouie) [inline pkg/paperwork/shipment_summary_worksheet.go]: Should these say "RM" instead of just "R"?

File: pkg/paperwork/forms.go (hunk 1 above)
Comment (mkrump) [reply pkg/paperwork/forms.go]: Indeed!

File: pkg/paperwork/shipment_summary_worksheet.go (hunk 2 above)
Comment (mkrump) [reply pkg/paperwork/shipment_summary_worksheet.go]: Rather than relying on defaults? I think that makes sense.

**Pull request merged**